common/src/main/scala/io/phdata/pulse/common/DocumentConversion.scala

```diff
@@ -17,11 +17,11 @@
 package io.phdata.pulse.common
 
 import io.phdata.pulse.common.domain.LogEvent
-import org.apache.solr.common.{ SolrDocument, SolrInputDocument }
+import org.apache.solr.common.{SolrDocument, SolrInputDocument}
 
 /**
- *
- */
+  *
```

Comment:
Empty comment?

Reply:
I think this was just the result of reformatting the code, it wasn't deliberate

Reply:
I see. Yes you can remove it.